docs/presto-admin-commands.rst

```diff
@@ -392,9 +392,18 @@ server install
 **************
 ::
 
-    presto-admin server install <local_path> [--nodeps]
-
-This command copies the presto-server rpm from ``local_path`` to all the nodes in the cluster, installs it, deploys the general presto configuration along with tpch connector configuration. The ``local_path`` should be accessible by ``presto-admin``.
+    presto-admin server install <rpm_specifier> [--rpm-source] [--nodeps]
+
+This command takes in a parameter ``rpm_specifier``, which must be a string describing the presto-server rpm to be used.
+The string can be 'latest'. This finds a local copy or downloads of the latest version of the presto rpm.
+The string can be a url. This finds a local copy or downloads the presto rpm found at the given url.
```

Comment:
local copy via a 'file:///' URL? If so, I'd suggest omitting that; it'll only confuse people who don't know about file:// URLs. People who know about them are probably savvy enough to figure them out themselves.

If not referring to file:// URLs, please explain further.